CRM/Case/BAO/Case.php

```diff
@@ -530,17 +537,27 @@ public static function getCaseActivityQuery($type = 'upcoming', $userID = NULL,
 
     if ($condition) {
       // CRM-8749 backwards compatibility - callers of this function expect to start $condition with "AND"
-      $query .= " WHERE (1) $condition ";
+      $query .= " WHERE (1) AND $condition ";
```

Comment:
the AND threw an error. see getNextScheduledActivity -- the $condition that's passed to this already has an AND prefixed, resulting in a double AND. actually -- that error occurred in the contact's case tab. removing the AND throws errors in the dashlet. so we need to do some code review to make sure $condition is passed to this function in a consistent way.

Reply:
Ahh ok, my bad will fix it and will check its usage on other places to avoid regression.